### ISSUE TITLE
Ensure all TODO comments are needed

### ISSUE THREAD

**Issue by JohnAlbin** [amazeeio-solutions/silverback-mono#459]
We have a bunch of TODO comments. And many of them are things that developers don't need _to do_ when starting a new project.

We should go through the list and anything that we need to fix in the starter should use FIXME instead.

All the TODO comments in be useful for a new project to read right away.

Additionally, we should change any `@TODO` and `@FIXME` to remove the `@` to make it consistent with the majority of our code comments.

In summary:
- Change all `@TODO` and `@FIXME` to `TODO` and `FIXME`, respectively.
- Change any TODO that are bugs/refactors that should be fixed in the starter to FIXME. And make sure there is a linked issue in this repo for all FIXMEs.
- Go through all TODO and ensure they are all things new projects **have to do**. Anything that new projects just need to keep in mind can just be regular code comments (no need for a TODO).
